README.md

```diff
@@ -1,5 +1,31 @@
 # RayFed
-A multiple parties joint, distributed execution engine on the top of Ray.
+A multiple parties joint, distributed execution engine based on Ray, to help build your own federated learning frameworks in minutes.
+
+
+## Features
+- **Ray Native Programming Pattern**  
+  
+  Let your write your federated and distributed computing applications like writing a single machine program.
```

Comment:
Let you write your federated and distributed computing applications like a single-machine program.
* "Let your" -> "Let you"
* "like writing a " -> "like a"
* (optional) "single machine" -> "single-machine"

Reply:
Great.